packages/frontend/src/utils/project/technology/get-data-availability-section.ts

```diff
@@ -8,15 +8,17 @@ import { getDaRisks } from '~/server/features/data-availability/utils/get-da-ris
 import { toTechnologyRisk } from '../risk-summary/to-technology-risk'
 
 export function getDataAvailabilitySection(project: Layer2 | Layer3) {
-  if (!project.dataAvailabilitySolution) {
+  // TODO: more than one bridge?
```

Comment:
Left over TODO? Leaving comment to make sure you're aware of it

Reply:
Aware of it. I want to somehow simplify the DAC descriptions but the current way it's structured makes it harder to do in one step. So you're left with this weird situation where an L2 could use multiple bridges (on the type level)

Reply:
I'd put in an assertion, so we're not blind to this hole